examples/data-sources/powerplatform_solution_checker_rules/data-source.tf

```diff
@@ -0,0 +1,35 @@
+terraform {
```

Comment:
rename file back from data-source.tf to datasource.tf

Reply:
the pattern seems to be data-source.  
![{E49A42F3-495B-4C8B-8FF3-0C1F1F1699C1}](https://github.com/user-attachments/assets/4a81b150-4585-437b-884b-301f617638e4)